0001-tests.patch

```diff
@@ -0,0 +1,140 @@
+From 1217055b0dfd9eda3418496633d92fce76b4dc00 Mon Sep 17 00:00:00 2001
```

Comment:
please delete the patch file. Add a *.patch rule in the .gitignore as well

Reply:
Updated accordingly.